<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deviceItemWidget</class>
 <widget class="QWidget" name="deviceItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="iconLabel">
     <property name="minimumSize">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:yellow;
border-radius:4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="deviceLabel">
     <property name="styleSheet">
      <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#CCCCCC;</string>
     </property>
     <property name="text">
      <string>设备A</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QLabel" name="stateLabel">
     <property name="styleSheet">
      <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:yellow;</string>
     </property>
     <property name="text">
      <string>已加载</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
